src/Benchmarks/Middleware/MultipleQueriesEfMiddleware.cs

```diff
@@ -21,20 +22,19 @@ public class MultipleQueriesEfMiddleware
         };
 
         private readonly RequestDelegate _next;
-        private readonly EfDb _db;
 
-        public MultipleQueriesEfMiddleware(RequestDelegate next, EfDb db)
+        public MultipleQueriesEfMiddleware(RequestDelegate next)
```

Comment:
It's possible the changes in this PR are correct, and the code in the dev branch was incorrectly caching too much in memory.  Either way, let's understand the difference and which is more representative of the intended scenario.